tests/bluetooth/tester/overlay-native.conf

```diff
@@ -1,3 +1,4 @@
 CONFIG_UART_PIPE=n
 CONFIG_SERIAL=y
 CONFIG_UART_NATIVE_POSIX=y
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
```

Comment:
This should probably be done for all existing LE Audio, Mesh and samples that enables BT_SETTINGS as well with their respective values of course, as to not change anything in those.

Reply:
Need to change here?

Reply:
No this is likely fine, but there's numerous LE Audio and Mesh samples that now may have insufficient stack sizes, as they went from 2048 to 1024, as well as the samples that used BT_SETTINGS that should have `CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2200`

Reply:
In other words, no samples should compile with the newly added warnings.

Reply:
@LingaoM Will you update the configs of existing samples that probably expected this stack size?

Reply:
I tried to search for other samples and found that other samples do not take effect under this condition（BT_RECV_IS_RX_THREAD=n）, including all bbsim.

As you say, add CMake warning for insufficient thread stack size can also provide way to let's user known it's should increase stack size when change default configuration.

Reply:
> I tried to search for other samples and found that other samples do not take effect under this condition（BT_RECV_IS_RX_THREAD=n）, including all bbsim.

That depends on the board being compiled though, e.g. when using the nRF53 with split architecture. 

> As you say, add CMake warning for insufficient thread stack size can also provide way to let's user known it's should increase stack size when change default configuration.

Exactly, and now the samples that enables BT_AUDIO, BT_MESH or BT_SETTINGS will get warnings when compiling for certain boards.

Reply:
Remove this line, since find already in https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bluetooth/tester/prj.conf#L3